repos/spack_repo/builtin/packages/cupla/package.py

```diff
@@ -1,47 +0,0 @@
-# Copyright Spack Project Developers. See COPYRIGHT file for details.
-#
-# SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-
-from spack_repo.builtin.build_systems.generic import Package
-
-from spack.package import *
-
-
-class Cupla(Package):
-    """C++ User interface for the Platform independent Library Alpaka"""
-
-    homepage = "https://github.com/alpaka-group/cupla"
-    git = "https://github.com/alpaka-group/cupla.git"
-    url = "https://github.com/alpaka-group/cupla/archive/refs/tags/0.3.0.tar.gz"
-
-    maintainers("vvolkl")
```

Comment:
@vvolkl Are you intending to add newer versions of cupla that build with non-deprecated versions of `alpaka`?